CMSgov/onemac
# Reset IDs for test automation

Story: https://qmacbis.atlassian.net/browse/OY2-18266
Endpoint: https://d2a1aljxsqid8d.cloudfront.net/

### Details
As we move to static IDs for our automated tests, we need a way to "reset" those IDs on deployment

### Changes
- created a resetData lambda which takes.a like of IDs and removes all records of those IDs from the one table (package) and the change requests table (submission)
- resetData runs after data migration
- resetData does NOT run on production branch

### Test Plan
1. login as a state submitter with Maryland access
2. Use some of the following IDs to create submissions:
Submission Dashboard, Medicaid SPA, Medicaid SPA RAI, CHIP SPA, CHIP SPA RAI:
"MD-22-0018",
  "MD-22-0019",
  "MD-22-0020",
  "MD-22-0021",
  "MD-22-0022",
  "MD-22-0023",
Submission Dashboard, Waiver "new" action
  "MD.12896",
  "MD.12958",
  "MD.38430",
Package Dashboard Base Waiver:
  "MD.33463.R00.00",
  "MD.39253.R00.00",
Package Dashboard Temp extension:
  "MD.33463.R00.TE00",
  "MD.33463.R00.TE01",
3. re-run deployment
4. Verify the records are removed from dashboard and database
